src/common/api/swagger.yaml

```diff
@@ -5915,6 +5921,7 @@ definitions:
       - 'belongs-to-majority-group'
       - 'valid-platform-network-settings'
       - 'ntp-synced'
+      - 'time-synced-between-host-and-service'
```

Comment:
Adding this constant requires an update of `src/common/config/constants`.
It already has a missing key, `service-has-sufficient-spoke-kube-api-access`, so ideally we can fix the two issues in this PR.

Reply:
Time passed, the API was updated within another PR so removing the commit here.